test/libc/calls/specialfile_test.c

```diff
@@ -70,6 +71,50 @@ TEST(specialfile, devNullRead) {
   ASSERT_SYS(0, 0, close(3));
 }
 
+TEST(specialfile, devRandomRead) {
+  char buf[8] = {0};
+  ASSERT_SYS(0, 3, open("/dev/random", O_RDONLY));
+  // qemu-aarch64 defines o_largefile wrong
```

Comment:
Seeing how many times this has to be written makes me want it to be factored out somehow. I dunno, maybe put the comment at the top of the file instead explaining why the individual tests are knocking out that bit?

Also I wonder if this is still relevant or if something better can be d one here, but I guess that's out of scope for this PR.